scss/_patterns_equal-height-row.scss

```diff
@@ -0,0 +1,155 @@
+@import 'settings';
+
+@mixin pseudo-border($width) {
+  background: var(--vf-color-border-default);
+  content: '';
+  display: block;
+  height: 1px;
+  position: absolute;
+  width: $width;
+}
+
+@mixin border-top {
+  border-top-color: $colors--theme--border-default;
+  border-top-style: solid;
+  border-top-width: 1px;
+  margin-top: -1px;
+}
+
+@mixin vf-p-equal-height-row {
+  .p-equal-height-row {
+    display: grid;
+    grid-template-columns: subgrid;
+    position: relative;
+
+    .p-equal-height-col {
+      display: grid;
+      grid-column: span calc($grid-columns / 4);
+      grid-row: span 10;
```

Comment:
This can't be auto because `.p-equal-height-col` is a subgrid and its parent grid has `grid-template-rows` set to default auto. This means the subgrid is actually defining the row structure for the parent grid, so we need to explicitly define the row span. Although we can set the span to 4 instead of 10 to cater for maximum 4 grid items, previously this is set to 10 to cater for an arbitrary number of grid items in a column, but we have decided to have maximum 4 items in a column now.